authentication_document.py

```diff
@@ -168,42 +170,55 @@ def parse_coverage(cls, _db, coverage, place_class=Place):
             # This library covers the entire universe! No need to
             # parse anything.
             place_objs.append(place_class.everywhere(_db))
-            coverage = dict()
+            coverage = dict() # Do no more processing
+
+        elif not isinstance(coverage, dict):
+            # The coverage is not in { nation: place } format.
+            # Convert it into that format using the default nation.
+            default_nation = place_class.default_nation(_db)
+            if default_nation:
+                coverage = {default_nation.abbreviated_name : coverage }
+            else:
+                # Oops, that's not going to work. We don't know which
+                # nation this place is in. Return a coverage object
+                # that makes it semi-clear what the problem is.
+                unknown["??"] = coverage
+                coverage = dict() # Do no more processing
 
-        for country, places in coverage.items():
```

Comment:
The rest of this code is just consistently using the term 'nation' instead of 'country'.
We say 'country' a couple other places in the code, and I'd eventually like to change it everywhere to match the constant we use for `Place.type`.